SCClassLibrary/Common/Control/Server.sc

```diff
@@ -367,22 +367,22 @@ Server {
 	}
 
 	initTree {
-		this.sendDefaultGroups;
-		tree.value(this);
-		Routine { ServerTree.run(this) }.play(AppClock);
+		forkIfNeeded {
```

Comment:
good idea.

by default, this won't use `AppClock`, so I am not sure this is intended. But you could pass a clock.

Reply:
Yes, thanks, should all be on AppClock. 
My current tests show that boot item order is still really unpredictable, 
so I would soon propose a first pass at a solution which follows #3286, 
with a single boot task, which does ServerBoot, s.tree, ServerTree, and 
a once-only list of doWhenBootedItems in order.

Reply:
ok!